spec/server/functional/courses.spec.coffee

```diff
@@ -0,0 +1,45 @@
+require '../common'
+utils = require '../utils'
+_ = require 'lodash'
+Promise = require 'bluebird'
+requestAsync = Promise.promisify(request, {multiArgs: true})
+
+describe 'GET /db/course', ->
+  beforeEach utils.wrap (done) ->
+    yield utils.clearModels([Course, User])
+    yield new Course({ name: 'Course 1' }).save()
+    yield new Course({ name: 'Course 2' }).save()
+    done()
+
+
+  it 'returns an array of Course objects', utils.wrap (done) ->
+    [res, body] = yield request.getAsync { uri: getURL('/db/course'), json: true }
+    expect(body.length).toBe(2)
+    done()
+
+describe 'GET /db/course/:handle', ->
+
+  beforeEach utils.wrap (done) ->
+    yield utils.clearModels([Course, User])
+    @course = yield new Course({ name: 'Some Name' }).save()
+    done()
+
+
+  it 'returns Course by id', utils.wrap (done) ->
+    [res, body] = yield request.getAsync {uri: getURL("/db/course/#{@course.id}"), json: true}
+    expect(res.statusCode).toBe(200)
+    expect(_.isObject(body)).toBe(true)
```

Comment:
Done. FYI you wrote those ;) https://github.com/codecombat/codecombat/blame/teacher-dashboard/spec/server/functional/courses.spec.coffee#L31

Not sure why it's showing up in my PR. From a merge, maybe..?